components/backline/plan.sh

```diff
@@ -0,0 +1,41 @@
+pkg_name=hab-backline
+pkg_origin=core
+pkg_version=$(cat ../../VERSION)
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('apachev2')
```

Comment:
Might as well touch this up to the correct SPDX-formatted string while you're at it.